src/schema/rules/errors.yaml

```diff
@@ -0,0 +1,252 @@
+---
+# This file describes rules for broad "standard" errors in the specification.
+
+# BIDS Validator Original Issue Code #26
+NiftiHeaderUnreadable:
+    code: NIFTI_HEADER_UNREADABLE
```

Comment:
Me curious: Why code isn't just the key of the entry?

Reply:
The idea was towards consistency, there are places in the schema that have custom error messages use the `code` and `message` keys:
https://github.com/bids-standard/bids-specification/blob/a93d4f2a27b2b79ba569a1d02f1e7f560af3525f/src/schema/rules/sidecars/asl.yaml#L27

Though those are nested in that `issue` key. I don't know if we want to consider lifting the `code` and `message` there a level higher or not. The schema needs its own schema.